src/Base/Support/Utils.php

```diff
@@ -244,4 +258,26 @@ public static function toRecursiveCollection(array $array): Collection
 
         return $collection;
     }
+
+    /**
+     * @param  null|int $flags
+     *
+     * @return string
+     */
+    private static function getFlagCase(?int $flags = null): string
+    {
+        if ($flags & Arrayable::CASE_SNAKE) {
+            return 'snake';
+        }
+
+        if ($flags & Arrayable::CASE_KEBAB) {
+            return 'kebab';
+        }
+
+        if ($flags & Arrayable::CASE_STUDLY) {
+            return 'studly';
+        }
+
+        return 'camel';
```

Comment:
Weet je zeker dat het ook echt Camel case is als je hier bent? Alternatief kan je hier ook een check doen op Camel case en aan het einde van de method een exception throwen als je het font niet herkend.